armory/utils/metrics.py

```diff
@@ -68,6 +101,127 @@ def l0(x, x_adv):
     return norm(x, x_adv, 0)
 
 
+class MetricList:
```

Comment:
Would it be useful to add an __iter__ method here if people are using this as a List?